doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst

```diff
@@ -0,0 +1,94 @@
+AD7134 IIO Application
+----------------------
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD7134 <https://www.analog.com/en/products/ad7134.html>`_ 
+* `AD4134 <https://www.analog.com/en/products/ad4134.html>`_ 
+
```

Comment:
Can Eval board links be added here as well.

Reply:
Relevant boards that can be used with this FW aren't released yet.

Reply:
Can we add a TODO then, like ad719x?

Reply:
A JIRA has been added to track this - https://jira.analog.com/browse/PCSW-557